rakelib/vendor.rake

```diff
@@ -124,6 +124,33 @@ namespace "vendor" do
   end # task gems
   task "all" => "gems"
 
+  task "deps", [:gemfile] => [ "dependency:bundler" ] do |task, args|
+    dependencies = {}
```

Comment:
if you're only going to pull the ".values" from `dependencies` then it should be an array or set